Content.Server/Drone/Components/DroneComponent.cs

```diff
@@ -13,6 +8,6 @@ public sealed class DroneComponent : Component
         [DataField("tools")] public List<EntitySpawnEntry> Tools = new();
         public List<EntityUid> ToolUids = new();
         public bool AlreadyAwoken = false;
-        public float InteractionBlockRange = 2.5f;
+        public float InteractionBlockRange = 2.15f;
```

Comment:
make this a datafield + 2.15 seems pretty arbitrary? why not just 2

Reply:
Since this is now the only interact check for non-items I wanted to make sure it's not exactly interact range or there might be some jank